src/helpers/sizeHelper.ts

```diff
@@ -0,0 +1,6 @@
+export function bytesToSize(bytes: number): string {
```

Comment:
```suggestion
export function formatBytes(bytes: number): string {
```
みたいな名前でも良いと思います（自分が書いたコードですが）

Reply:
この関数、結局使ってないんですよね...消すのを忘れていました...
とりあえずこの先も使わなさそうなので消しちゃおうと思います...!